source/objects/waveClass.m

```diff
@@ -65,6 +65,7 @@
         waveAmpTime1 = [];  % Wave elevation time history at a wave gauge 1 location specified by user [m] 
         waveAmpTime2 = [];  % Wave elevation time history at a wave gauge 2 location specified by user [m] 
         waveAmpTime3 = [];  % Wave elevation time history at a wave gauge 3 location specified by user [m] 
+        waveGaugeExist = [false, false, false]; % Boolean array to track if wave gauges have been specified (loc ~= 0,0)
         A = [];             % Wave amplitude [m]. For regular waves or 2*(wave spectrum vector) for irregular waves

```

Comment:
Instead of creating a new property within the waveClass I recommend checking to see if each if any of the wave gauges have been initialized, for example`if true(waves.wavegauge1loc ~= [nan,nan])`. This removes the middle step of defining a new property, and helps keep the class a little cleaner/simpler.

Reply:
Also, I think we should change the way the wave gauge locations are initialized. Instead of setting them to `[0,0]`, I think we should set them to `wavegauge1loc = [nan,nan];`